crates/rust-test-input/src/lib.rs

```diff
@@ -1,14 +1,6 @@
+#![no_std]
+
+/// Adds two numbers together.
 pub fn add(left: u64, right: u64) -> u64 {
     left + right
 }
-
-#[cfg(test)]
-mod tests {
-    use super::*;
-
-    #[test]
-    fn it_works() {
-        let result = add(2, 2);
-        assert_eq!(result, 4);
-    }
-}
```

Comment:
Because it wasn't really testing anything; what we actually will end up with is a harness that runs something in Rust and in the compiled rep and checks correctness between the two. This was just from the default cargo library template.